recipes/tasks.py

```diff
@@ -0,0 +1,12 @@
+# Create your tasks here
+from __future__ import absolute_import, unicode_literals
+from celery import shared_task
+from django.core.mail import send_mail
+from HipoProject.settings import EMAIL_HOST_USER
+from django.template.loader import render_to_string
+
+
+@shared_task
+def send_email_wrapper(subject, recipient_list, context):
```

Comment:
`send_html_email` can be a good name. What do you think ?
`send_email` is too generic name for this function because it only sends registration email.
Let's pass `template_name` as a parameter to this function. So we can use `send_html_email` or a similar name.